libs/designsystem/src/lib/components/tab-navigation/tab-navigation-item/tab-navigation-item.component.scss

```diff
@@ -0,0 +1,78 @@
+@use '@kirbydesign/core/src/scss/utils';
+
+$tab-item-text-max-width: 100px;
+$bottom-border-height: 1px;
+$border-color-standard: utils.get-color('medium');
+$border-color-selected: utils.get-color('dark');
+$divider-max-width-breakpoint: utils.$page-content-max-width + (utils.size('s') * 2);
+
+:host {
+  position: relative;
+  padding-bottom: $bottom-border-height * 2;
+
+  button {
+    font-family: var(--kirby-font-family);
+    background-color: var(--kirby-button-background-color, initial);
+    color: var(--kirby-button-color, inherit);
```

Comment:
Consider not exposing these as CSS Custom Props, as it seems like it is not needed right now. So for now it could just use our tokens, e.g. `utils.get-color('background-color');` or whatever color fits.

Reply:
Done

Reply:
Done